index.html

```diff
@@ -11,12 +11,12 @@
     <script type="text/javascript" src="script.js" defer></script>
 </head>
 <body>
-  <div id="instructionModal" class="modal">
+<div id="instructionModal" class="modal">
     <div class="modal-content">
-        <div class="introClose">
-            <div></div>
+        <div class="instructionTop">
+            <div class="leftX"></div>
             <h1>Instructions!</h1>
```

Comment:
h1 is probably not the right tag to use here. this title isnt the most important title on the page